packages/react-core/src/components/NotificationDrawer/examples/NotificationDrawerLightweight.tsx

```diff
@@ -0,0 +1,185 @@
+import React from 'react';
+import {
+  Button,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  EmptyStatePrimary,
+  EmptyStateVariant,
+  NotificationDrawer,
+  NotificationDrawerBody,
+  NotificationDrawerHeader,
+  NotificationDrawerGroup,
+  NotificationDrawerGroupList,
+  NotificationDrawerList,
+  NotificationDrawerListItem,
+  NotificationDrawerListItemBody,
+  NotificationDrawerListItemHeader,
+  Title
+} from '@patternfly/react-core';
+import SearchIcon from '@patternfly/react-icons/dist/esm/icons/search-icon';
+
+export const NotificationDrawerLightweight: React.FunctionComponent = () => {
+  const [firstGroupExpanded, setFirstGroupExpanded] = React.useState(false);
+  const [secondGroupExpanded, setSecondGroupExpanded] = React.useState(true);
+  const [thirdGroupExpanded, setThirdGroupExpanded] = React.useState(false);
+
+  const toggleFirstDrawer = (_event: any, value: boolean) => {
+    setFirstGroupExpanded(value);
+    setSecondGroupExpanded(false);
+    setThirdGroupExpanded(false);
+  };
+
+  const toggleSecondDrawer = (_event: any, value: boolean) => {
+    setSecondGroupExpanded(value);
+    setFirstGroupExpanded(false);
+    setThirdGroupExpanded(false);
+  };
+
+  const toggleThirdDrawer = (_event: any, value: boolean) => {
+    setThirdGroupExpanded(value);
+    setSecondGroupExpanded(false);
+    setFirstGroupExpanded(false);
+  };
+
+  const onDrawerClose = () => {
+    // do something cool in a callback
+  };
+
+  return (
+    <NotificationDrawer>
+      <NotificationDrawerHeader onClose={onDrawerClose} />
+      <NotificationDrawerBody>
+        <NotificationDrawerGroupList>
+          <NotificationDrawerGroup
+            title="First notification group"
+            isExpanded={firstGroupExpanded}
+            count={4}
+            isRead={false}
```

Comment:
Here, for the NotificationDrawerGroup component, read/isRead for openshift / lightweight is not used, so we always set isRead to true so as not to show the blue unread pill. Rather the counts should always be grey as pictured.
<img width="434" alt="Screen Shot 2022-08-22 at 11 21 21 AM" src="https://user-images.githubusercontent.com/35978579/185957972-5a51e92a-fa1d-4fa8-84a6-070ab56d9ae6.png">

Reply:
got it, thanks for the review @jcaianirh